src/components/AddActor.jsx

```diff
@@ -1,44 +1,33 @@
-import React, {useState} from 'react';
+import React, { useState } from 'react';
+import { newActor } from './api';
 
-const AddActor = ({actors, setActors}) => {
+const AddActor = ({ actors, setActors }) => {
   const [addActorError, setAddActorError] = useState('');
 
   const newActorHandler = async (e) => {
     e.preventDefault();
-
-    const newActor = e.target.elements.actorName.value;
-    // Add the new actor to the database, returning the new actor object
-    try {
-      const response = await fetch('/api/actors', {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify({ name: newActor }),
-      });
-
-      if (response.ok) {
-        const actor = await response.json();
-        // add the actor to the actor list and reset values
-        setActors([...actors, actor]);
-        setAddActorError('');
-        e.target.reset();
-      } else {
-        setAddActorError('Error when adding the actor');
-      }
-    } catch (error) {
-      setAddActorError(error.message);
+    const name = e.target.elements.actorName.value;
+    // Add the new actor values to the database
+    const result = await newActor(name);
+    if (result.success) {
+      // add the actor to the actor list and reset values
```

Comment:
`reset values` -> what values?

Reply:
I added a hopefully more clear comment explaining that it's resetting the input value and the error message.